container-core/src/main/java/com/yahoo/container/core/config/testutil/HandlersConfigurerTestWrapper.java

```diff
@@ -143,8 +144,12 @@ protected void configure() {
     }
 
     private static class SimpleContainerThreadpool implements ContainerThreadPool {
+
         private final Executor executor = Executors.newCachedThreadPool();
-        @Override public Executor executor() { return executor; }
```

Comment:
I don't agree with this style change. The original style is more compact without impacting readability (imho).

Reply:
I agree with you actually but nobody else seem to like it so I've given up :-)

Reply:
Why change it then? 😛

Reply:
If I'm the only one who wants some style I'll go for consistency instead.

Reply:
FYI I have seen other Vespa devs using that style as well btw. Same applies to my indentation preference for parameter lists.

Reply:
Yes I think Valeri also like the wrong indentation style and you're both batshit crazy.
Let me change the @overrides back now that I know I have an ally on that :-)